src/libsyntax/mut_visit.rs

```diff
@@ -309,6 +309,10 @@ pub trait MutVisitor: Sized {
     fn visit_span(&mut self, _sp: &mut Span) {
         // Do nothing.
     }
+
+    fn visit_impl_item_kind(&mut self, iik: &mut ImplItemKind) {
```

Comment:
Looks like this change is not necessary, the function is introduced but not used.